### PR TITLE
Enable Jenkins CI

### PR DESCRIPTION
**Pull request by andyliuliming**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    stages {
+        stage('rubocop') {
+            steps {
+                sh 'gem install rubocop'
+                sh 'src/bosh_azure_cpi/bin/rubocop_check'
+            }
+        }
+        stage('unit-test') {
+            steps {
+                sh 'src/bosh_azure_cpi/bin/unit-test'
```
Comment (bingosummer) [inline Jenkinsfile]: Why not use `test-unit` directly?

File: scripts/install_packages.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env bash
```
Comment (bingosummer) [inline scripts/install_packages.sh]: Why we need this script?

File: src/bosh_azure_cpi/bin/rubocop_check [hunk 3]
```diff
@@ -2,9 +2,16 @@
 which rubocop > /dev/null 2>&1
 [[ "$?" -eq "1" ]] && {
   echo "You need to install rubocop using 'gem install rubocop'"
-  exit
+  exit 1
 }
+export XDG_CACHE_HOME=$(pwd)
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 pushd $DIR/..
+ls
```
Comment (bingosummer) [inline src/bosh_azure_cpi/bin/rubocop_check]: How about adding 2 blanks indent in the block `pushd`?

File: src/bosh_azure_cpi/bin/test-unit [hunk 4]
```diff
@@ -3,15 +3,12 @@
 args=$@
 [[ "$#" != "1" ]] && args=spec/unit/*
 
-pushd $SRC_DIR
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```
Comment (bingosummer) [inline src/bosh_azure_cpi/bin/test-unit]: SRC_DIR is defined in `.envrc` in the root directory of CPI. It needs `direnv`. I guess, in the CI, `direnv` is not installed. So, you replace it with `DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"`. Correct?

Comment (andyliuliming) [reply src/bosh_azure_cpi/bin/test-unit]: yes do not want to make it depend on the direnv.

File: scripts/install_packages.sh (hunk 2 above)
Comment (andyliuliming) [reply scripts/install_packages.sh]: no need , will delete this.

File: Jenkinsfile (hunk 1 above)
Comment (andyliuliming) [reply Jenkinsfile]: ok, will merge these to one using the args.

**Pull request merged**